guides/nuxtjs-starter-guide/configuring.mdx

```diff
@@ -0,0 +1,102 @@
+---
+_schema: guide-item
+_uuid: "cfe55ac8-e5ce-4960-8336-8624b28a5893"
+_created_at: "2023-06-22 06:54:28 +0000"
+title: "Configuring"
+nav_title: "Configuring"
+order: 4
+published: true
+image: "/documentation/static/CloudCannonDocumentationog.jpg"
+description: "This guide will walk through the steps required to get your Nuxt site built, editable and live on CloudCannon."
+tags: []
+related_articles: null
+related_links: null
+---
+
+Some SSGs, like Jekyll, Hugo, and Eleventy, have strong opinions about how to organize the content in your repo.
+CloudCannon can run automatic processes on these SSGs and provide basic editing out of the box.
+Nuxt takes a less opinionated approach and allows you to decide where and how content is stored.
+This means that for Nuxt sites, CloudCannon out of the box only has the source editor available.
+
+#### Interlude: Editing terminology
+
+CloudCannon has a few concepts that make organizing data easier:
+
+* **Data**\: These are standalone data files. This is ideal for site configuration or files that don't need to repeat. Data is edited by altering a single file.
+* **Collections**\: A folder of files that represent a repeatable data format. This is a good fit for pages, blogs, staff members, recipes, etc. Collections are edited by altering any file or adding more files.
+
+Using the [Nuxt Content](https://content.nuxtjs.org/) module makes integrating your Nuxt site with CloudCannon easy.
+This module has clear anologues to the organization of data mentioned above.
+Most importabtly, it requires that you separate your site content from the templating and rendering logic.
+This makes it easiest to integrate the site into the CMS.
+
+When using the Nuxt Content module, we recommend grouping related content in directories.
+For instance, if your site has a blog with multiple posts, it's a good idea to keep those posts in a `posts/` folder.
+
+<comp.Notice info_type="info">
+  CloudCannon does not support Server-side Rendering as our build and hosting environments are decoupled.
+</comp.Notice>
+
+CloudCannon supports a set of file formats for files in Data or Collections:
+
+* Structured data files: JSON, YAML, TOML, CSV, TSV
+* Markup files: HTML, Markdown
+* Combination files: HTML with front matter, Markdown with front matter
+
+Front matter refers to a section at the top of a markup file that contains structured data. CloudCannon supports JSON, YAML and TOML front matter. Markdown comes in many flavours, so be sure to checkout configuring your Markdown engine.
+
+*<u><strong>If you need another supported format, please contact support and we would be happy to look into it.</strong></u>*
```

Comment:
This could maybe be a Notice? 🤷

Reply:
Hmm I agree, but it's the same across all the other starter guides and I can't be bothered changing them all in this PR